R/helpers.R

```diff
@@ -94,6 +94,11 @@ measureTime = function(expr, ee = parent.frame()) {
   proc.time()[[3L]] - before
 }
 
+attachTrainingTime = function(model, task, data) {
+  model$times = data[, getTaskTargetNames(task)[1L]]
```

Comment:
Is this the model as returned by the learner, e.g. a coxph object?

Why do you not use `attachTrainingInfo()`?

Reply:
Is this an attribute like is used in `attachTrainingInfo()`? It is a vector of times...